src/coreclr/pal/src/eventprovider/lttngprovider/CMakeLists.txt

```diff
@@ -63,9 +63,19 @@ add_dependencies(coreclrtraceptprovider generated_eventing_headers)
 find_library(LTTNG NAMES lttng-ust)
 
 target_link_libraries(coreclrtraceptprovider
-                      ${LTTNG}
+                    ${LTTNG}
 )
 
+if(CLR_CMAKE_TARGET_ARCH_LOONGARCH64)
```

Comment:
Unless this results in build breakage, I would move changes from this file to https://github.com/dotnet/runtime/pull/62887 that way each PR would be self-contained.

Reply:
Thanks.
Had moved.